packages/lustre-v6/lustre-v6.6.101.7/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "The Lustre V6 Verimag compiler"
+description: """
+This package contains:
+  - lv6: the (current) name of the compiler (and interpreter via -exec)
+  - the lustre-v6 ocaml lib: allows to call the Lustre v6 interpreter from ocaml
+  - the lustre-v6 rdbg plugin: allows to debug Lustre v6 program wth rdbg.
+"""
+maintainer: "erwan.jahier@univ-grenoble-alpes.fr"
+authors: "Erwan Jahier and Pascal Raymond"
+license: "CeCILL"
+homepage: "http://www-verimag.imag.fr/lustre-v6.html"
+bug-reports: "http://www-verimag.imag.fr/lustre-v6.html"
+depends: [
+  "ocaml" {>= "4.05"}
+  "base-unix"
+  "extlib" {build} | "extlib-compat" {build}
+  "lutils" {>= "1.49"}
+  "oasis" {build & >= "0.4.7"} | "oasis-mirage" {build & >= "0.4.7"}
+  "ocamlbuild" {build}
+  "ocamlfind"
```

Comment:
Is this a build dependency or you depend on `findlib` ar runtime?

Reply:
I'm not 100% sure. Maybe it does as this package contains a dynamic lib (lustre-v6.cmxs).
This dependancy was generated by oasis2opam (which might not be a proof that it is
correct w.r.t. the need of findlib at runtime.

Reply:
No problem, let's keep it as it is